Crash uploading media

It seems that `develop` is broken on Media Uploading.
The app crashes if you try to upload a media in the editor(s) or in the Media Library.
Below is the stack trace of the error:
```
03-12 10:41:31.280 16210-16210/org.wordpress.android E/AndroidRuntime: FATAL EXCEPTION: main
                                                                       Process: org.wordpress.android, PID: 16210
                                                                       org.greenrobot.eventbus.EventBusException: Invoking subscriber failed
                                                                           at org.greenrobot.eventbus.EventBus.handleSubscriberException(EventBus.java:505)
                                                                           at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:487)
                                                                           at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:479)
                                                                           at org.greenrobot.eventbus.HandlerPoster.handleMessage(HandlerPoster.java:67)
                                                                           at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                           at android.os.Looper.loop(Looper.java:135)
                                                                           at android.app.ActivityThread.main(ActivityThread.java:5221)
                                                                           at java.lang.reflect.Method.invoke(Native Method)
                                                                           at java.lang.reflect.Method.invoke(Method.java:372)
                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:898)
                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:693)
                                                                        Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'float java.lang.Float.floatValue()' on a null object reference
                                                                           at org.wordpress.android.ui.uploads.PostUploadNotifier.getCurrentMediaProgress(PostUploadNotifier.java:664)
                                                                           at org.wordpress.android.ui.uploads.PostUploadNotifier.getCurrentOverallProgress(PostUploadNotifier.java:650)
                                                                           at org.wordpress.android.ui.uploads.PostUploadNotifier.updateNotificationProgress(PostUploadNotifier.java:640)
                                                                           at org.wordpress.android.ui.uploads.PostUploadNotifier.updateNotificationProgressForMedia(PostUploadNotifier.java:631)
                                                                           at org.wordpress.android.ui.uploads.UploadService.onMediaUploaded(UploadService.java:847)
                                                                           at java.lang.reflect.Method.invoke(Native Method)
                                                                           at java.lang.reflect.Method.invoke(Method.java:372)

```

### Steps to reproduce the behavior
- Open the Editor
- Add a picture to it

##### Tested on Wiko Ridge 4G, Android 5


cc @mzorz

--- a/WordPress/src/main/java/org/wordpress/android/ui/uploads/PostUploadNotifier.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/uploads/PostUploadNotifier.java
@@ -661,7 +661,8 @@ class PostUploadNotifier {
         float currentMediaProgress = 0.0f;
         int size = sNotificationData.mediaItemToProgressMap.size();
         for (int i = 0; i < size; i++) {
-            float itemProgress = sNotificationData.mediaItemToProgressMap.get(i);
+            int key = sNotificationData.mediaItemToProgressMap.keyAt(i);
+            float itemProgress = sNotificationData.mediaItemToProgressMap.get(key);
             currentMediaProgress += (itemProgress / size);
         }
         return currentMediaProgress;

